main.go

```diff
@@ -423,7 +423,18 @@ func run() (err error) {
 	return
 }
 
+// Handle panic if the crash occurs during mounting.
+func handlePanic() {
+	// Detect if panic happens in main go routine.
+	a := recover()
+	if a != nil {
+		logger.Info("Panic: ", a)
```

Comment:
why is panic an info message?

Reply:
As panic is will do exit code 1, before that we are printing the cause of panic.

Reply:
message type - info or error represents the type of log. Yes panic will crash the process, that doesn't mean we print the source of panic as info message. 
https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels

FYI, here you made panic log as fatal: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels

Reply:
Yes I thought that but [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/fs/wrappers/error_mapping.go#L93) em errormapping logger have Fatal method but our [logger](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/logger/logger.go) does not have Fatal method.

Should I add that method in our logger in proceed ?

Reply:
yes